grpc4bmi/bmi_julia_model.py

```diff
@@ -0,0 +1,615 @@
+from typing import List
+
+from bmipy import Bmi
+import numpy as np
+from juliacall import Main as jl
+
+def install(package):
+    """Add package to Julia environment.
+    
+    Args:
+        package: Name of package to install.
+    """
+    jl.Pkg.add(package)
+
+class BmiJulia(Bmi):
+    """Python Wrapper of a Julia based implementation of BasicModelInterface.
+
+    BasicModelInterface is available in https://github.com/Deltares/BasicModelInterface.jl repo.
+
+    Args:
+        package: Name of Julia package which contains interface and model classes
+        implementation_name: Name of Julia variable which implements BasicModelInterface
+        model_name: Name of Julia model class 
+
+    """
+    def __init__(self, package, implementation_name, model_name):
+        self.module = package
+        self.model_name = model_name
+        jl.seval("using " + package)
+        self.model = getattr(getattr(jl, self.module), self.model_name)
```

Comment:
What if someone has `JuliaModel.bmi.Bmi`? In the Wflow.jl testing I had a recursive getattr function.

E.g. `model_name="model.Model"` and `implementation_name="bmi.Bmi"`

Reply:
using seval instead of getattr